src/modules/commander/Commander.hpp

```diff
@@ -347,6 +347,7 @@ class Commander : public ModuleBase<Commander>, public ModuleParams
 
 	unsigned int	_leds_counter{0};
 
+	manual_control_setpoint_s	_prev_manual_control_setpoint{};	///< the previous manual control setpoint
```

Comment:
It seems quite confusing to have two last/previous manual_control_setpoints in commander that are saved at different points in time.